common/install_miopen.sh

```diff
@@ -90,7 +90,10 @@ MIOPEN_CMAKE_COMMON_FLAGS="
 -DMIOPEN_BUILD_DRIVER=OFF
 "
 
-if [[ $ROCM_INT -ge 50000 ]]; then
+if [[ $ROCM_INT -ge 50100 ]]; then
+    MIOPEN_CMAKE_DB_FLAGS="-DMIOPEN_EMBED_DB=gfx900_56;gfx906_60;gfx90878;gfx90a6e;gfx1030_36"
+    MIOPEN_BRANCH="release/rocm-rel-5.1-staging"
+elif [[ $ROCM_INT -ge 50000 ]]; then
     MIOPEN_CMAKE_DB_FLAGS="-DMIOPEN_EMBED_DB=gfx900_56;gfx906_60;gfx90878;gfx90a6e;gfx1030_36"
     MIOPEN_BRANCH="release/rocm-rel-5.0-staging"
 elif [[ $ROCM_INT -ge 40500 ]]; then
```

Comment:
@jeffdaily I vote for removing older ROCm versions which are not in the list here: https://github.com/pytorch/builder/pull/993/commits/7486d1656325f0cc140ab4b7f05413b7958e9099#diff-4016942f0012705eb74a286c992b334bdca8388994fe0115ab30ffd577334c5bR50

Thoughts? The only reason to keep them around is if we foresee building wheels with those older ROCm versions locally, but the chances of that seem pretty bleak.